retriever/lib/socrata.py

```diff
@@ -129,8 +129,8 @@ def find_socrata_dataset_by_id(dataset_id):
                 print("No socrata dataset exists for the id : {dataset_id}".format(
                     dataset_id=dataset_id))
 
-    except HTTPError as e:
-        print("HTTPError : ", e)
+    except (HTTPError,requests.exceptions.ConnectionError) as e:
+        print("Error Occured :", e)
```

Comment:
A space `(HTTPError,requests.exception`  `(HTTPError, requests.exception`
```Python
print("Error Occured :", e)
print("Error Occured : ", e) # Add a space after : 
```

Reply:
@henrykironde I have made the corrections. Please check it. Is there anything else I should consider?